src/components/Footer.css

```diff
@@ -0,0 +1,15 @@
+.App-footer {
```

Comment:
as the footer is it's own component, `.Footer `would be enough. :)
& because it is the component class, you also use Capitalization then.

Reply:
i agree on that.

Reply:
thanks ! changing it. naming is tricky :D